pyproject.toml

```diff
@@ -7,6 +7,8 @@ license = "GPL-3.0-or-later"
 
 [tool.poetry.dependencies]
 python = "^3.8"
+ansible = "~=2.10.5"
+boto = "~=2.49.0"
```

Comment:
Is this old `boto` version required somewhere by the Ansible stuff?

Reply:
Yes 😬